packages/core/src/components/checkbox/checkbox.scss

```diff
@@ -0,0 +1,71 @@
+@use '../../helpers';
+@use '../../internal';
+@use '../../mixins';
+@use '../included';
+
+@mixin Checkbox() {
+  $stroke-size: helpers.space(0.25);
+
+  @if not included.is-included('indicator-container') {
```

Comment:
💅🏻  [nitpick]

I'd name it something cheeky but clearer like
```scss
@if not is.included('indicator-container');
```

```scss
@if not includes.the('indicator-container');
```

```scss
@if not already.included('indicator-container');
```

Or something less cheeky.

```scss
@if not internal.included('indicator-container');
```

I'd also just name it `indicator`, since it has styles for more than the container.

Reply:
I don't want to have "internal" in the name, as I plan also to use the same for other cases, e.g. when one component requires another component (not internal one).

Also, would not want to go with just `indicator`, because internal component is called `IndicatorContainer` - I think better to match the name.